server/channels/app/channel.go

```diff
@@ -187,6 +187,7 @@ func (a *App) JoinDefaultChannels(c request.CTX, teamID string, user *model.User
 
 		message := model.NewWebSocketEvent(model.WebsocketEventUserAdded, "", channel.Id, "", nil, "")
 		message.Add("user_id", user.Id)
+		message.Add("user", user)
```

Comment:
Seems like the user sanitization has been removed. Why?

Reply:
sorry my bad let me check again.

Reply:
I have fixed it. Sorry for my carelessness. 

Because I found out that the "channel.go" file was deleted from redesigning structure.  I was afraid of copying the whole old file to the new "channel.go" file.

by the way, I am so sorry again.

Reply:
No problem. To err is human :smile: That is what the code reviews are for.

Reply:
thank you, guys. I appreciate it.

One more question.
Now, I am coding this task "https://github.com/mattermost/mattermost-server/issues/21193" "React"+"Golang". So, I have to move to code in "mattermost-server" repo, don't I ?

Thank you again.

Reply:
@nongpond03 Yes. Webapp and Server code now live in the same repo. So you can code the golang part and the react part in one single repo (`mattermost-server`).